pkgs/tools/admin/winbox/default.nix

```
@@ -36,7 +39,7 @@ let
       ${wine}/bin/wineboot -u
     fi
 
-    ${wine}/bin/wine ${executable} "$@"
+    ${wine}/bin/${wineExecutable} ${executable} "$@"
```

```suggestion
    ${wine}/bin/${wine.meta.mainProgram} ${executable} "$@"
```

(and drop `wineExecutable` again)

---

I don't know how that thing is working, but I'm glad to have learned something new :)

---

I just tested everything and it works. Where is the "wine64" string is generated ? Looks like black magic to me now :D

---

It's from here:
https://github.com/NixOS/nixpkgs/blob/58930f0b7f3171d398fbc7b297ff0a8eb18de537/pkgs/applications/emulators/wine/base.nix#L201
Which is populated in here for wine64:
https://github.com/NixOS/nixpkgs/blob/58930f0b7f3171d398fbc7b297ff0a8eb18de537/pkgs/applications/emulators/wine/packages.nix#L44
The default being here again:
https://github.com/NixOS/nixpkgs/blob/58930f0b7f3171d398fbc7b297ff0a8eb18de537/pkgs/applications/emulators/wine/base.nix#L9